telemetry-proxy/telemetry_proxy.go

```diff
@@ -120,4 +177,6 @@ func websocketHandler(w http.ResponseWriter, r *http.Request) {
 			break
 		}
 	}
+	// Remove the connection from the map when it closes
+	defer delete(connections, conn)
```

Comment:
Also, shouldn't this defer be moved up to line 164 (where it is inserted into the connections map)?

Reply:
Good point. Moved there